source/client/HandView.py

```diff
@@ -20,16 +31,22 @@ def __init__(self, controller):
         pygame.init()
         # initialize hand_info
         # (hand_info =UICardWrapped elements of current_hand).
+        self.current_hand = []
         self.hand_info = []
-        self.wrapped_hand = []
-        # create window for game - left side is table=PUBLIC, right side is users
-        # TO DO - change this so bottom is current hand and top is table.
-        hand_disp_width = UIC.Disp_Width * UIC.Hand_Col_Fraction
+        # create window for game - top is table=PUBLIC, bottom is users view of hand.
+        # not used: hand_disp_height = UIC.Disp_Width * UIC.Hand_Row_Fraction
```

Comment:
If they aren't used they can die, if they are going to be used then this comment is temporary and maybe instead note how they will be used. A not used comment is generally not something clear enough to want to put it into shared code.